.travis.yml

```diff
@@ -33,6 +33,10 @@ cache:
 before_cache:
   - rm -f $HOME/.cache/pip/log/debug.log
 
+before_install:
+  - sudo apt-get update -qq
+  - sudo apt-get install -y unrar
+
```

Comment:
Unrar is quite a strong dependency, how does subliminal behaves with legendastv provider if unrar is missing?